src/wallet/mod.rs

```diff
@@ -5414,4 +5417,40 @@ pub(crate) mod test {
         // ...and checking that everything is fine
         assert_fee_rate!(psbt, details.fee.unwrap_or(0), fee_rate);
     }
+
+    #[cfg(feature = "hardware-signer")]
+    #[test]
+    fn test_create_signer() {
```

Comment:
Maybe this test should be ignored by default? Since it depends on external setup

Reply:
Or alternatively you can add a separate feature to test hardware signers, and if explicitly enabled we run this test